.happy/config.json

```diff
@@ -0,0 +1,14 @@
+{
+    "config_version": "v1",
+    "terraform_version": "0.13.5",
+    "default_env": "rdev",
+    "app": "hosted-cellxgene",
+    "default_compose_env": ".env.ecr",
+    "environments": {
```

Comment:
We don't need to define environments for dev, stage, and prod?

Reply:
All happy config in explorer is currently for rdev purposes only.